### PR TITLE
feat: added banner && COC

### PR DESCRIPTION
**Pull request by RickDeb2004**

### CODE REVIEW

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🚀

File: src/components/marginals/Navbar/styles.js [hunk 1]
```diff
@@ -126,6 +134,17 @@ export const LogoContainer = styled.div`
     h-[47.663px]
     flex-shrink-0
 `}
+  @media (max-width: 720px) {
+    //   display: flex;
```
Comment (ashutosh-rath02) [inline src/components/marginals/Navbar/styles.js:138]: Remove comments

File: src/components/marginals/Navbar/styles.js [hunk 2]
```diff
@@ -217,9 +236,23 @@ export const CustomLogo = styled.div`
   &::after {
     display: none;
   }
+
   @media (max-width: 1024px) {
     width: 48px;
   }
+  @media (max-width: 720px) {
+    //position: absolute;
```
Comment (ashutosh-rath02) [inline src/components/marginals/Navbar/styles.js:244]: Remove comments

File: src/components/marginals/Navbar/styles.js [hunk 3]
```diff
@@ -229,3 +262,15 @@ export const CustomLink = styled.a`
   font-weight: bold;
   margin-top: 2rem;
 `;
+// export const Wrapper = styled.div`
```
Comment (ashutosh-rath02) [inline src/components/marginals/Navbar/styles.js:265]: Remove unnecessary files

**Review by ashutosh-rath02 — APPROVED** [review]

**Review by Shurtu-gal — APPROVED** [review]

**Pull request merged**